crates/nu-command/src/filters/drop/column.rs

```diff
@@ -121,14 +123,18 @@ fn drop_cols(
                             drop_record_cols(val, head, &drop_cols)?
                         }
                     }
-                    Ok(Value::list(vals, span).into_pipeline_data())
+                    Ok(Value::list(vals, span)
+                        .into_pipeline_data()
```

Comment:
I think ones like this should probably be `.into_pipeline_data_with_metadata(metadata)`

Reply:
I did it like that at first but then I saw it being done the other way in few places so I changed it for consistency. I can change it back

Reply:
ok. Where possible, it should be `.into_pipeline_data_with_metadata(metadata)` but in some places it isn't possible.